dialogues/target_detection_dialog.py

```diff
@@ -240,15 +276,59 @@ def get_image_from_hdr(self):
         img = cv2.cvtColor(np_img, cv2.COLOR_BGR2RGB)
         
         return img
+    
+    def generate_result_images(self):
```

Comment:
img is used instead of image through most of your code, keep one wording

Reply:
Fixed